common/src/uds_fe_programming.rs

```diff
@@ -75,38 +79,45 @@ impl UdsFeProgrammingFlow {
     // |           UDS Region                |
     // |          (64 bytes)                 |
     // +-------------------------------------+
-    //
-    // FE Partitions, separate region in fuse controller memory handled by MCU, but at same base as UDS
-    //
-    // +-------------------------------------+ <- uds_seed_dest_base_addr_low()
+    // +-------------------------------------+
+    // |        UDS Digest                   |
+    // |         (8 bytes)                   |
+    // +-------------------------------------+
+    // +-------------------------------------+ <- FE Base = uds_seed_dest_base_addr_low() + UDS Region + UDS Digest
     // |        FE Partition 0               |
     // |         (8 bytes)                   |
-    // +-------------------------------------+ <- base + (0 * 16)
+    // +-------------------------------------+ <- FE Base + 8
     // |     FE Partition 0 Digest           |
     // |         (8 bytes)                   |
-    // +-------------------------------------+ <- base + (0 * 16) + 8
+    // +-------------------------------------+ <- FE Base + (1 * 16)
```

Comment:
I think this is not accurate for 2.1, as there are zeroization fuses as well: https://github.com/chipsalliance/caliptra-ss/blob/main/src/fuse_ctrl/doc/otp_ctrl_mmap.md